app/Component/Form/FormFields.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import EditActions from '../../Actions/EditActions';
+import { InputField } from '../Field';
+
+class FormFields extends React.Component {
+    buildFields(entry) {
+        let fields = [];
+
+        for (let i in this.props.fields) {
+            let field = this.props.fields[i];
+            let fieldName = field.name();
+            let renderedField;
+            let value = this.props.values.get(fieldName);
+            let className = 'show-value react-admin-field-' + field.name() + ' ' +
+                (field.getCssClasses(entry) || 'col-sm-10 col-md-8 col-lg-7');
+
+            switch (field.type()) {
+                case 'string':
+                    renderedField = <InputField type={'text'} name={fieldName} value={value} updateField={this.props.updateField} />;
```

Comment:
The link between the field name and the component should be handled in a FieldView, just like in ng-admin